devtools/create_project/xcode.cpp

```diff
@@ -25,6 +25,7 @@
 
 #include <fstream>
 #include <algorithm>
+#include <sys/stat.h>
```

Comment:
I guess I also don’t know how to get cross-platform code for finding a directory into create_project. I tried to include common/fs.cpp but it ends up requiring dependencies of irrelevant things like GUI::EventRecorder because it uses flags for the entire build. I don’t know about how to make make do what I want it to do, and I don’t even know if this approach would be acceptable anyway.